Header header
octomap_msgs/Octomap[] octomaps
string owner
uint16 num_octomaps

